### PR TITLE
[CAIRO-2018] Adding event request for Cairo 2018

### PR DESCRIPTION
**Pull request by aameen79**
As per our meeting with Kris, this is the pull request sent, I hope it's done right :)

### CODE REVIEW

**Comment by mattstratton** [comment]
This is missing all of the content files; I'll try to fix it up tomorrow.

**Comment by aameen79** [comment]
All content files added now and fixed

**Comment by mattstratton** [comment]
It’s not showing up in the event list in the deploy preview - I’ll take a look in a few hours to see if I can figure out why.

**Comment by MikeRosTX** [comment]
Without a confirmed venue location, **startdate** and **enddate** need to be removed so it appears under **TBD** list.

**Comment by aameen79** [comment]
I updated the location in 2018-cairo.yml @mattstratton @MikeRosTX

**Comment by MikeRosTX** [comment]
Need to include the **ITIDA building - Smart Village** address on the **location.md** page as well.

File: data/events/2018-cairo.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "30.079061,31.019447" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Cairo" # Defaults to city, but you can make it the venue name.
+
+location_address: "ITIDA building - Smart Village" #Optional - use the street address of your venue. This will show up on the welcome page if set.
```
Comment (bridgetkromhout) [inline data/events/2018-cairo.yml]: This doesn't work because it's not a street address. You need the actual address here if you want Google Maps to work.

**Pull request merged**